fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ChunkWriter.java

```diff
@@ -38,6 +38,7 @@
  */
 public class ChunkWriter extends AbstractItemWriter {
     private static final Logger logger = Logger.getLogger(ChunkWriter.class.getName());
+    AmazonS3 cosClient = null;
```

Comment:
Just confirming:  ChunkWriter get re-instantiated for each job, right?

Reply:
yes